src/cloud/project/project-upgrade.md

```diff
@@ -107,7 +107,7 @@ To upgrade the Magento version:
 1. Change to your Magento root directory and set the upgrade version using the [version constraint syntax].
 
    ```bash
-   composer require "magento/magento-cloud-metapackage":">=2.x.2 <2.x.3" --no-update
+   composer require "magento/magento-cloud-metapackage":">=2.3.5 <2.3.6" --no-update
```

Comment:
Change the version number in the code sample to use placeholders for current and next version--same technique used in the [Upgrade to ece-tools](https://devdocs.magento.com/cloud/project/ece-tools-upgrade-project.html#metapackage) topic.

```suggestion
   composer require "magento/magento-cloud-metapackage":">=current_version <next_version" --no-update
```



On line 114, there is a note: 

```   You must use the version constraint syntax to successfully update the `{{site.data.var.ct}}` package.```

Update that note to add info about where they can find the most current version constraint: 

```You must use the version constraint syntax to successfully update the `{{site.data.var.ct}}` package. You can always find the latest version constraint in the [`magento-cloud` template](https://github.com/magento/magento-cloud/blob/master/composer.json).```

Reply:
**implemented**